src/main/java/fr/uge/dummychargepoint/socket/ConfigurationClient.java

```diff
@@ -0,0 +1,47 @@
+package fr.uge.dummychargepoint.socket;
+
+import org.java_websocket.client.WebSocketClient;
+import org.java_websocket.handshake.ServerHandshake;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.net.URI;
+import java.util.Map;
+
+public class ConfigurationClient extends WebSocketClient implements ConfigurationApplication {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigurationClient.class);
+
+  public ConfigurationClient(URI serverUri, Map<String, String> httpHeaders) {
```

Comment:
Map<HttpHeader, String> httpHeaders maybe? It would be more verbose but more explicit
HttpHeader would be an enum

Reply:
Yep, you're right, but here in the futur, there is only one header needed for the ocpp handshake.

Should we create an enum for only one data ?

Reply:
And also, it is used by the parent WebSocketClient so we must leave it like that